packaging/zip/start.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
```

Comment:
Shouldn't we use common start.sh?

Reply:
Not now, it will be done in future tickets